Firebase/Auth/Source/Public/FIRAuthDataResult.h

```diff
@@ -24,8 +24,8 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /** @class FIRAuthDataResult
-    @brief Helper object that contains the result of a successful sign-in, link and reauthenticate.
-        It contains a reference to a @c FIRUser and @c FIRAdditionalUserInfo.
+    @brief Helper object that contains the result of a successful sign-in, link and reauthenticate
+        action. It contains a reference to an instance of @c FIRUser and @c FIRAdditionalUserInfo.
```

Comment:
could be rephrased as "It contains references to a FIRUser instance and a FIRAdditionalUserInfo instance."